pre_commit/clientlib.py

```diff
@@ -153,9 +153,9 @@ def ordered_load_normalize_legacy_config(contents):
     data = ordered_load(contents)
     if isinstance(data, list):
         # TODO: Once happy, issue a deprecation warning and instructions
-        return collections.OrderedDict([('repos', data)])
-    else:
-        return data
+        data = collections.OrderedDict([('repos', data)])
+
+    return data
```

Comment:
fwiw this style was intentional -- it's very easy for a human to see at a glance that there are two disparate paths that the code could take.  By decomposing the `if` / `else`, there's overhead to parsing the code.

Reply:
Interesting view, thanks.